kits/agp1/scripts/deploy_agp1.js

```diff
@@ -0,0 +1,120 @@
+const namehash = require('eth-ens-namehash').hash
+
+const getAccounts = require('@aragon/os/scripts/helpers/get-accounts')
+const deployDAOFactory = require('@aragon/os/scripts/deploy-daofactory.js')
+const logDeploy = require('@aragon/os/scripts/helpers/deploy-logger')
+
+const getEventResult = (receipt, event, param) => receipt.logs.filter(l => l.event == event)[0].args[param]
+const getAppProxy = (receipt, id, index=0) => receipt.logs.filter(l => l.event == 'InstalledApp' && l.args.appId == id)[index].args.appProxy
+
+const apps = ['finance', 'vault', 'voting']
+const appIds = apps.map(app => namehash(require(`@aragon/apps-${app}/arapp`).environments.default.appName))
+
+const globalArtifacts = this.artifacts // Not injected unless called directly via truffle
+const globalWeb3 = this.web3 // Not injected unless called directly via truffle
+const defaultOwner = process.env.OWNER
+const defaultENSAddress = process.env.ENS
+const defaultDAOFactoryAddress = process.env.DAO_FACTORY
+const defaultMinimeTokenAddress = process.env.MINIME_TOKEN
+
+module.exports = async (
+  truffleExecCallback,
+  {
+    artifacts = globalArtifacts,
+    web3 = globalWeb3,
+    owner = defaultOwner,
+    ensAddress = defaultENSAddress,
+    daoFactoryAddress = defaultDAOFactoryAddress,
+    minimeTokenAddress = defaultMinimeTokenAddress,
+    verbose = true
+  } = {}
+) => {
+  const kitName = 'AGP1Kit'
+
+  const log = (...args) => {
+    if (verbose) { console.log(...args) }
+  }
+
+  if (!owner) {
+    const accounts = await getAccounts(web3)
+    owner = accounts[0]
+    log('OWNER env variable not found, setting APM owner to the provider\'s first account')
+  }
+
+  log(`${kitName} with ENS ${ensAddress}, owner ${owner}`)
+
+  const TokenFactoryWrapper = artifacts.require('TokenFactoryWrapper')
+  const DAOFactory = artifacts.require('DAOFactory')
+  const ENS = artifacts.require('ENS')
+
+  if (!ensAddress) {
+    errorOut('ENS environment variable not passed, aborting.')
```

Comment:
I'm not sure, but are we missing an import or a definition for `errorOut()`? Seems like all the kits actually use this without a definition... is it injected globally by truffle?

Reply:
I have no idea, the first time I saw it was when bingen added it to the kit's code.

Reply:
:astonished: haha, sorry, I'll remove it then